forc-plugins/forc-client/src/cmd/deploy.rs

```diff
@@ -7,8 +7,13 @@ pub use forc_util::tx_utils::Salt;
 
 use crate::NodeTarget;
 
+forc::cli_examples! {
+    [ Deploy a single contract => deploy " bc09bfa7a11a04ce42b0a5abf04fd437387ee49bf4561d575177e2946468b408" => r#".*Error making HTTP request.*"# ]
+    [ Deploy a single contract from a different path => deploy " bc09bfa7a11a04ce42b0a5abf04fd437387ee49bf4561d575177e2946468b408 --path ../tests/" => r#".*Error making HTTP request.*"# ]
```

Comment:
```suggestion
    [ Deploy a single contract => deploy "bc09bfa7a11a04ce42b0a5abf04fd437387ee49bf4561d575177e2946468b408" => r#".*Error making HTTP request.*"# ]
    [ Deploy a single contract from a different path => deploy "bc09bfa7a11a04ce42b0a5abf04fd437387ee49bf4561d575177e2946468b408 --path ../tests/" => r#".*Error making HTTP request.*"# ]
```

Are these spaces meant to be here?